### ISSUE TITLE
Adding discernible text to open label info icon 

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#1158]
**Description**:
The information icon on the open data label doesn't have any discernable text. This means that this element can not been seen by screen readers. This fails WCAG 4.1.2 

![image](https://user-images.githubusercontent.com/83226114/157840121-cfd7a883-01a9-486c-81f9-64fae150efb0.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done

**Comment by joshuadkitenge** [comment]
This issue should be addressed in #904
